build-constraints.yaml

```
@@ -4283,8 +4283,8 @@ packages:
         - earcut
         - vector-circular
         # required by reanimate as of 0.4.2.0
-        - hgeometry < 0 # via vector-0.12.2.0 https://github.com/commercialhaskell/stackage/issues/5886
-        - hgeometry-combinatorial < 0 # via vector-0.12.2.0 https://github.com/commercialhaskell/stackage/issues/5886
+        - hgeometry # via vector-0.12.2.0 https://github.com/commercialhaskell/stackage/issues/5886
+        - hgeometry-combinatorial # via vector-0.12.2.0 https://github.com/commercialhaskell/stackage/issues/5886
```

```suggestion
        - hgeometry
        - hgeometry-combinatorial
```
nit: These comments are here to note why a bound was added, so they can be removed when the bound is removed.